File: VB/SaveLoadDefinition/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
' ...

Namespace SaveLoadDefinition
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		' A temporary path to save a report to.
		Private filePath As String = "C:\Temp\XtraReport1.repx"

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a report.
			Dim report As New XtraReport1()

			' Save the report's layout to the configuration file.
			report.SaveLayout(filePath)
		End Sub

		Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button2.Click
			' Create a report.
			Dim report As New XtraReport1()

			' Load the report's layout from the specified file.
			report.LoadLayout(filePath)

			' Display the print preview.
			report.ShowPreview()
		End Sub

	End Class
End Namespace

File: CS/SaveLoadDefinition/Form1.cs
using DevExpress.XtraReports.UI;
using System;
using System.Windows.Forms;
// ...

namespace SaveLoadDefinition {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        // A temporary path to save a report to.
        string filePathCodeDom = @"C:\Temp\XtraReport1.repx";

        private void SaveRepx_Click(object sender, EventArgs e) {
            // Create a report.
            XtraReport1 report = new XtraReport1();

            // Save the report's layout to the configuration file.
            report.SaveLayout(filePathCodeDom);
        }

        private void LoadRepx_Click(object sender, EventArgs e) {
            // Create a report.
            XtraReport1 report = new XtraReport1();

            // Load the report's layout from the specified file.
            report.LoadLayout(filePathCodeDom);

            // Display the print preview.
            ReportPrintTool printTool = new ReportPrintTool(report);
            printTool.ShowPreviewDialog();
        }

        string filePathXml = @"C:\Temp\XtraReport1.xml";

        private void SaveXml_Click(object sender, EventArgs e) {
            // Create a report.
            XtraReport1 report = new XtraReport1();

            // Save the report's layout to the configuration file.
            report.SaveLayoutToXml(filePathXml);
        }

        private void LoadXml_Click(object sender, EventArgs e) {
            // Create a report.
            XtraReport1 report = new XtraReport1();

            // Load the report's layout from the specified file.
            report.LoadLayoutFromXml(filePathXml);

            // Display the print preview.
            ReportPrintTool printTool = new ReportPrintTool(report);
            printTool.ShowPreviewDialog();
        }

    }
}